src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java

```diff
@@ -376,6 +374,26 @@ public boolean canAdministerProject(String projectID, String organizationId) {
         return false;
     }
 
+    public boolean canAdministerProject(FeedSource feedSource) {
+        return canAdministerProject(feedSource.projectId, feedSource.organizationId());
+    }
+
+    public boolean canAdministerProject(com.conveyal.datatools.manager.models.Project project) {
+        return canAdministerProject(project.id, project.organizationId);
+    }
+
+    public boolean canAdministerProject(Label label) {
```

Comment:
I think it'd make more sense to have this method correspond to the model:

```suggestion
    public boolean canAdministerLabel(Label label) {
```

Reply:
Skipping due to explanation in https://github.com/ibi-group/datatools-server/pull/410#discussion_r683720627